Plutarch/DataRepr/Internal.hs

```diff
@@ -203,10 +275,29 @@ pmatchDataSum d handlers =
 -}
 newtype PIsDataReprInstances (a :: PType) (s :: S) = PIsDataReprInstances (a s)
 
+-- Q: Why are these superclass constraints here?
 class (PMatch a, PIsData a) => PIsDataRepr (a :: PType) where
```

Comment:
`PMatch a` should be replaced with `PlutusType a`. This is to make sure people don't forget to derive all necessary instances. `PIsData a` helps avoid accidentally unsound code.

Reply:
Sounds good. Updated!